SetupDataPkg/Tools/CommonUtility.py

```diff
@@ -65,6 +86,21 @@ def value_to_bytearray(value, length):
     return bytearray(value_to_bytes(value, length))
 
 
+def bytes_to_bracket_str(bytes):
+    return '{ %s }' % (', '.join('0x%02x' % i for i in bytes))
+
+
+def array_str_to_value(val_str):
+    val_str = val_str.strip()
+    val_str = strip_delimiter(val_str, '{}')
+    val_str = strip_quote(val_str)
+    value = 0
+    for each in val_str.split(',')[::-1]:
+        each = each.strip()
+        value = (value << 8) | int(each, 0)
+    return value

```

Comment:
With the cdata stuff gone, do we actually use any of these methods anymore?

Reply:
This specific function is needed when editing the numbers. But the other methods below are removed.